### PR TITLE
Added a period to the end of all alt-text strings

### PR DESCRIPTION
**Pull request by mikehoffms**
Ran script:
https://review.docs.microsoft.com/en-us/help/contribute/contribute-accessibility-multimedia?branch=master#cleanup-script-to-add-periods-to-alt-text
That fixed both types of image tagging: Markdown and :::

### CODE REVIEW

File: microsoft-edge/devtools-guide-chromium/css/inspect.md [hunk 1]
```diff
@@ -51,19 +51,19 @@ The **Accessibility** section of the **Inspect** overlay displays information ab
...
+![Elements that lack sufficient contrast have a warning icon..](images/inspect-tool-bad-contrast.msft.png)
 
 
 <!-- ====================================================================== -->
 ## Inspecting non-accessible elements
 
 Elements that have the CSS property of `pointer-events: none` aren't available to the **Inspect** tool.  In the [Inspect Demo](https://microsoftedge.github.io/DevToolsSamples/inspector/inspector-demo.html) page, hover over the `Overlay Button` and you will see that the parent element (`div.wrapper`) is shown instead of the `Overlay Button`.
 
-![An element that has a CSS property of 'pointer events: none' can't be selected.](images/inspect-tool-element-element-without-pointer-events.msft.png)
+![An element that has a CSS property of 'pointer events: none' can't be selected..](images/inspect-tool-element-element-without-pointer-events.msft.png)
```
Comment (captainbrosset) [inline microsoft-edge/devtools-guide-chromium/css/inspect.md:62]: There are 2 periods here. Looks like the script failed.

File: microsoft-edge/devtools-guide-chromium/css/inspect.md [hunk 2]
```diff
@@ -51,19 +51,19 @@ The **Accessibility** section of the **Inspect** overlay displays information ab
 
 For example, in the [Inspect Demo](https://microsoftedge.github.io/DevToolsSamples/inspector/inspector-demo.html) page, for the `Bad Contrast` button, the **Inspect** overlay has a warning icon next to the contrast value of 1.77.  The **Inspect** overlay also shows that the button isn't focusable via keyboard.  The button can't be navigated to via keyboard, because the button is implemented as a `<div>` element with a class of `button`, instead of being implemented as a `<button>` element.
 
-![Elements that lack sufficient contrast have a warning icon.](images/inspect-tool-bad-contrast.msft.png)
+![Elements that lack sufficient contrast have a warning icon..](images/inspect-tool-bad-contrast.msft.png)
```
Comment (captainbrosset) [inline microsoft-edge/devtools-guide-chromium/css/inspect.md:54]: There are 2 periods here. Looks like the script failed.

File: microsoft-edge/devtools-guide-chromium/css/inspect.md [hunk 3]
```diff
@@ -51,19 +51,19 @@ The **Accessibility** section of the **Inspect** overlay displays information ab
...
 
-![An element that has a CSS property of 'pointer events: none' can't be selected.](images/inspect-tool-element-element-without-pointer-events.msft.png)
+![An element that has a CSS property of 'pointer events: none' can't be selected..](images/inspect-tool-element-element-without-pointer-events.msft.png)
 
 To inspect elements that have the CSS property of `pointer-events: none`, press `Shift` while hovering over the element.  There's also a color overlay on page layout regions, indicating that you are in an advanced selection mode.
 
-![Pressing the Shift key while selecting elements on the page allows you to select elements that have a CSS style property of 'pointer-events: none'.](images/inspect-tool-with-shift.msft.png)
+![Pressing the Shift key while selecting elements on the page allows you to select elements that have a CSS style property of 'pointer-events: none'..](images/inspect-tool-with-shift.msft.png)
```
Comment (captainbrosset) [inline microsoft-edge/devtools-guide-chromium/css/inspect.md:66]: And here too

File: microsoft-edge/devtools-guide-chromium/css/inspect.md [hunk 4]
```diff
@@ -75,7 +75,7 @@ When you click an element in the rendered page:
 *  The corresponding DOM node is highlighted.
 *  The **Styles** tool shows the CSS that's applied to the element.
 
-![When you click an element in the rendered page, the Styles tool shows the styles that are applied to the element.](images/inspect-tool-highlighted-styles.msft.png)
+![When you click an element in the rendered page, the Styles tool shows the styles that are applied to the element..](images/inspect-tool-highlighted-styles.msft.png)
```
Comment (captainbrosset) [inline microsoft-edge/devtools-guide-chromium/css/inspect.md:78]: Here too

**Review by captainbrosset — APPROVED** [review]
Looks great, thanks for running the script.
I only found a few instances where there were 2 periods instead of 1. I wonder if the script has a bug where it sometimes doesn't detect that a period is already present.

File: microsoft-edge/devtools-guide-chromium/css/inspect.md (hunk 4 above)
Comment (mikehoffms) [reply microsoft-edge/devtools-guide-chromium/css/inspect.md:78]: Apparently there's a bug in the script for Markdown, it doesn't check to see if already a .
Did a S/R on:    `..]` to globally fix.

**Comment by mikehoffms** [comment]
> Looks great, thanks for running the script. I only found a few instances where there were 2 periods instead of 1. I wonder if the script has a bug where it sometimes doesn't detect that a period is already present.

Confirmed it's a bug, a known bug that's been reported.  Globally fixed the result.

**Pull request merged**